### ISSUE TITLE
clean up the ENABLE_JS flag 

### ISSUE THREAD

**Issue by Pomax** [mozilla/learning.mozilla.org#2138]
with the build rewrite and no longer generating a giant static site, the ENABLE_JS flag (which doesn't determine whether to run with or without JS integrally, but only whether to run the app as pure JS app or as static HTML without spawning JS) may simply be no longer relevant.

Once #2133 lands we should look at whether we can get rid of it entirely

**Comment by Pomax** [comment]
@2133 has landed.
